src/main/kotlin/com/atlassian/performance/tools/virtualusers/api/VirtualUserResult.kt

```diff
@@ -29,7 +30,31 @@ class VirtualUserResult internal constructor(
      *
      * @since 3.12.0
      */
-    fun streamMetrics(): Stream<ActionMetric> {
+    @Deprecated("Use more specific name: streamActions", replaceWith = ReplaceWith("streamActions()"))
+    fun streamMetrics(): Stream<ActionMetric> = streamActions()
+
+    /**
+     * Each VU executes a scenario. Scenario contains actions. Each action can emit multiple metrics.
+     * This streams these scenario metrics emitted by this VU.
+     * A metric can be surrounded by another one:
+     * e.g. a `SEND_MAIL` metric can contain `LOAD_EDITOR`, `FILL_FORM`, `COMPOSE_MAIL` and `SEND` metrics.
+     * There can be time gaps between metrics, e.g. when the action does some processing or VU is diagnosing/throttling.
+     *
+     * @since 3.13.0
+     */
+    fun streamActions(): Stream<ActionMetric> = stream(actionMetrics)
+
+    /**
+     * Each VU has performs one task at a time, e.g scenario actions, diagnosing, throttling.
+     * They should not overlap on a timeline.
+     *
+     * @since 3.13.0
+     */
+    fun streamTasks(): Stream<ActionMetric> = stream(taskMetrics)
```

Comment:
In earlier PR versions, this was called `streamActivityMetrics`. But I found it confusing, because there were too many `act*` words:
- VU actions
- VU activity
- VU "Act" activity

Now we have:
- VU actions
- VU tasks
- VU "Act" task